src/js/binaryen.js-post.js

```diff
@@ -576,19 +576,19 @@ function wrapModule(module, self = {}) {
   // 'callIndirect', 'returnCall', 'returnCallIndirect' are deprecated and may
   // be removed in a future release. Please use the the snake_case names
   // instead.
-  self['callIndirect'] = self['call_indirect'] = function(target, operands, params, results) {
+  self['callIndirect'] = self['call_indirect'] = function(target, table, operands, params, results) {
```

Comment:
It would be good to test the changes to the JS API in the kitchen-sink.js test.

Reply:
Updated the js tests to always pass "0" as the table name to both `call_indirect` and `return_call_indirect`.